controllers/discordactions.js

```diff
@@ -141,9 +143,36 @@ const updateDiscordImageForVerification = async (req, res) => {
   }
 };
 
+const markUnverified = async (req, res) => {
+  try {
+    const [usersInRdsDiscordServer, allRdsLoggedInUsers] = await Promise.all([getDiscordMembers(), fetchAllUsers()]);
+    const rdsUserMap = {};
```

Comment:
i don't see this rdsUserMap is not cleared anywhere, that mean's this is grow as user(s) will login

Reply:
As per the discussion, the map won't increase as this code works for existing users only, new users automatically get that role applied.